spec/registrant_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true)
+    @registrant_2 = Registrant.new('Penny', 15)
+  end
+
+  describe '#initialize' do
+    it 'can initialize' do
+      expect(@registrant_1).to be_an_instance_of(Registrant)
+      expect(@registrant_2).to be_an_instance_of(Registrant)
+    end
+
+    it 'can set name' do
+      expect(@registrant_1.name).to eq('Bruce')
+      expect(@registrant_2.name).to eq('Penny')
+    end
+
+    it 'can set age' do
+      expect(@registrant_1.age).to eq(18)
+      expect(@registrant_2.age).to eq(15)
+    end
+
+    it 'can set permit status' do
+      expect(@registrant_1.permit).to be true
+      expect(@registrant_2.permit).to be false
```

Comment:
For this one I _would_ test this for both registrants since we're expecting different outcomes, one with the default value and one being set.